plugins/chat/assets/javascripts/discourse/models/chat-channel.js

```diff
@@ -128,54 +128,6 @@ export default class ChatChannel {
     this.currentUserMembership.lastViewedAt = new Date();
   }
 
-  findIndexOfMessage(id) {
```

Comment:
I decided for now to move all these functions to the manager only, it was always weird to wonder if messages operations were going to be able on the main model (channel/thread) or if you had to go to the manager.